### PR TITLE
Add React client to Express Server

### PR DESCRIPTION
**Pull request by kellyho9021**
Created React client project within Express server project to handle front end of the app
Successfully verified connection between client and server
Updated Readme file

### CODE REVIEW

File: .idea/Baby-Sharks-Room.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
```
Comment (hvuong91) [inline .idea/Baby-Sharks-Room.iml:2]: Isn't this auto-generated from IDEA? .idea should be added to the .gitignore

File: package.json [hunk 2]
```diff
@@ -2,14 +2,20 @@
   "name": "baby-sharks-room",
   "version": "0.0.0",
   "private": true,
+  "main": "server.js",
   "scripts": {
-    "start": "node ./bin/www",
-    "dev": "nodemon ./bin/www"
+    "client-install": "cd client && npm install",
+    "start": "node server.js",
```
Comment (hvuong91) [inline package.json]: Please put back bin/www back as this hides the actual starting js file. This yields more abstraction.

www should have the port set up. Listening port can be removed in server.js itself. The correct reference in www

```
var app = require('../server');
```

File: server.js [hunk 3]
```diff
@@ -3,16 +3,21 @@ var express = require('express');
...
 
 var indexRouter = require('./routes/index');
 var usersRouter = require('./routes/users');
-
+var reactClient = require('./routes/reactClient')
+var cors = require('cors');
 var app = express();
 
+//set port 5000
+app.listen(port, () => console.log(`Listening on port ${port}`));
```
Comment (hvuong91) [inline server.js]: This can be removed regarding the reference to bin/www comment

**Comment by hvuong91** [comment]
All packages can be updated to the latest, ex. concurrently is at 5.2.0 while the referenced version is at 3.5.1

File: server.js (hunk 3 above)
Comment (kellyho9021) [reply server.js]: I tried to remove it and it failed to start the server

File: server/server.js [hunk 4]
```diff
@@ -6,13 +6,17 @@ var logger = require('morgan');
 
 var indexRouter = require('./routes/index');
 var usersRouter = require('./routes/users');
-
+var reactClient = require('./routes/reactClient')
+var cors = require('cors');
 var app = express();
+var port = 5000;
```
Comment (hvuong91) [inline server/server.js:12]: Unused variable?

File: client/src/serviceWorker.js [hunk 5]
```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
```
Comment (hvuong91) [inline client/src/serviceWorker.js:1]: No action: This file can be removed later as we currently don't use it.

**Pull request merged**